forte/ontology_specs/ag_news.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "ag_news",
+    "imports": [
+      "base_ontology.json"
+    ],
+    "definitions": [
+      {
+        "entry_name": "ft.onto.ag_news.Article",
+        "parent_entry": "ft.onto.base_ontology.Document",
+        "attributes": [
+            {
+              "name": "class_id",
```

Comment:
I don't think we should have a `class_id`. We can derive this id from the name, so this information is a bit redundant.

Reply:
Okay. I will just store the class names in the config, then.